_posts/2016-08-28-mouse-dna-magnetic-fields.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: No permanent damage to mouse DNA from low magnetic fields
+author: RaoOfPhysics
+image:
+  feature: 2016-08-28-mouse-dna-magnetic-fields.jpg
+  credit: Nogas1974
+  caption: “DNA com GGN”
+---
+
+_Sustained exposure not found to cause permanent damage in cells in brain, kidney and liver that are involved in the transport, reabsorption or storage of iron._
```

Comment:
Again, more active, and perhaps shorter?
"Brain, liver, and kidney cells can withstand sustained exposure"